saleor/graphql/extensions/resolvers.py

```diff
@@ -1,20 +1,56 @@
-import graphene
-
+from ...extensions.base_plugin import BasePlugin
 from ...extensions.manager import get_extensions_manager
-from ..utils import sort_queryset
+from ...extensions.models import PluginConfiguration
+from .filters import filter_plugin_search
 from .sorters import PluginSortField
-from .types import Plugin
 
 
-def resolve_plugin(info, plugin_global_id):
+def resolve_plugin(info, plugin_name):
     manager = get_extensions_manager()
-    plugin = graphene.Node.get_node_from_global_id(info, plugin_global_id, Plugin)
+    plugin: BasePlugin = manager.get_plugin(plugin_name)
     if not plugin:
         return None
-    return manager.get_plugin_configuration(plugin.name)
+    return PluginConfiguration(
+        id=plugin.PLUGIN_NAME,
+        active=plugin.active,
+        configuration=plugin.configuration,
+        description=plugin.PLUGIN_DESCRIPTION,
+        name=plugin.PLUGIN_NAME,
+    )
 
 
 def resolve_plugins(sort_by=None, **_kwargs):
+    plugin_filter = _kwargs.get("filter", {})
     manager = get_extensions_manager()
-    qs = manager.get_plugin_configurations()
-    return sort_queryset(qs, sort_by, PluginSortField)
+    sort_field = (
+        sort_by.get("field", PluginSortField.NAME) if sort_by else PluginSortField.NAME
+    )
+    sort_reverse = sort_by.get("direction", False) if sort_by else False
+    if sort_field == PluginSortField.IS_ACTIVE:
+
+        plugins = sorted(
+            manager.plugins,
+            key=lambda p: (not p.active if sort_reverse else p.active, p.PLUGIN_NAME),
+        )
+    else:
+        plugins = sorted(manager.plugins, key=lambda p: p.PLUGIN_NAME)
+        if sort_reverse:
+            plugins = reversed(plugins)
```

Comment:
IMO this could go to a function to make the resolver more readable. Here we could have then e.g. `sort_plugins(manager.plugins, sort_by)`.